website/profile/views.py

```diff
@@ -412,8 +413,10 @@ def oauth_application_detail(auth, **kwargs):
         raise HTTPError(http.GONE)
 
     app_detail_url = api_v2_url("applications/{}/".format(client_id))  # Send request to this URL
+    app_reset_url = api_v2_url("applications/{}/reset/".format(client_id))
     return {"app_list_url": '',
-            "app_detail_url": app_detail_url}
+            "app_detail_url": app_detail_url,
+            "app_reset_url": app_reset_url}
```

Comment:
Instead of passing the reset URL into the view, fetch it from the `data.links.reset` field of the detail payload. (similar to how we handle the deletion URL in the detail view) 

This ensures that there is one authoritative source for all information about the instance, and URLs don't need to be updated in multiple places.